Makefile

```diff
@@ -14,11 +15,18 @@ deps:
 bash:
 	# Don't run if you don't have to. Adds several megs to repo with every commit.
 	rm -rf $(BASH_DIR) && mkdir -p $(BASH_DIR)/linux $(BASH_DIR)/osx
-	curl -Ls https://github.com/robxu9/bash-static/releases/download/4.3.30/bash-linux \
-		> $(BASH_DIR)/linux/bash
-	curl -Ls https://github.com/robxu9/bash-static/releases/download/4.3.30/bash-osx \
+
+	curl -#SLk https://github.com/robxu9/bash-static/releases/download/$(BASH_STATIC_VERSION)/bash-linux \
+		> $(BASH_DIR)/linux/bash && strip $(BASH_DIR)/linux/bash
+
+	curl -#SLk https://github.com/robxu9/bash-static/releases/download/$(BASH_STATIC_VERSION)/bash-osx \
 		> $(BASH_DIR)/osx/bash
-	chmod +x $(BASH_DIR)/**/*
```

Comment:
i removed the double star ** here since this is only available when shopt globstar is enabled